### PR TITLE
Fix broken links from sandbox moved files

### LINKED ISSUE #2949: Dead links in v0.18 docs
From: https://knative.slack.com/archives/C93E33SN8/p1603222516244600

> I'm trying to upgrade my cluster to 0.18.0 and it looks like the help page points to a couple of bad URLs:
Source page:
https://knative.dev/docs/install/any-kubernetes-cluster/
Dead Links:
https://github.com/knative/eventing-contrib/blob/v0.18.0/natss/config/broker/README.md
kubectl apply --filename https://github.com/knative/eventing-contrib/releases/download/v0.18.0/natss-channel.yaml (edited)

### PR DESCRIPTION
**Pull request by abrennan89**
Fixes #2949 and additional broken links not reported

Related issues:
#2970

### CODE REVIEW

File: docs/install/any-kubernetes-cluster.md [hunk 1]
```diff
@@ -587,13 +587,12 @@ kubectl apply --filename {{< artifact repo="eventing" file="in-memory-channel.ya
 {{% tab name="NATS Channel" %}}
 
 1. First, [Install NATS Streaming for
-   Kubernetes](https://github.com/knative/eventing-contrib/blob/{{<
-   version >}}/natss/config/broker/README.md)
+   Kubernetes](https://github.com/knative-sandbox/eventing-natss/blob/master/config/broker/README.md)
```
Comment (abrennan89) [inline docs/install/any-kubernetes-cluster.md:590]: actually https://github.com/knative-sandbox/eventing-natss/tree/master/config maybe

**Review by pierDipi — COMMENTED** [review]
- Line 549: 
  - org: knative-sandbox 
  - repo: eventing-kafka 
  - file: channel-consolidated.yaml
- Line 798
  - org: knative-sandbox 
  - repo: eventing-kafka 
  - file: source.yaml
- Line 841
  - https://github.com/knative-sandbox/eventing-couchdb/blob/master/source/README.md

**Review by pierDipi — COMMENTED** [review]
in `docs/docs/eventing/channels/channels-crds.md`
from:
```
[KafkaChannel](https://github.com/knative/eventing-contrib/tree/{{< branch >}}/kafka/channel/README.md)
```
to:
```
[KafkaChannel](https://github.com/knative-sandbox/eventing-kafka/tree/{{< branch >}}/pkg/channel/consolidated/README.md)
```

**Comment by pierDipi** [comment]
> TODO: Update https://knative.dev/docs/eventing/samples/writing-event-source/#target-audience eventing-contrib -> knative-sandbox

I guess you're referring to:

> This tutorial is for having your event source in an independent repository. If you would like to contribute Knative’s eventing-contrib and have your source there, there are some instructions at Adding the event source to eventing-contrib.

I don't know what would be the recommendation to contribute a new source since without eventing-contrib people needs a new repo and there is a well-defined path that involves the TOC.

/cc @knative/eventing-wg-leads

**Comment by pierDipi** [comment]
I think the page https://knative.dev/docs/eventing/samples/writing-event-source/07-eventing-contrib/ isn't useful anymore and can be removed.

**Comment by pierDipi** [comment]
Kafka pieces shouldn't be backported to 0.18, so we should create a different PR for what can be backported and what not. (?)

**Comment by abrennan89** [comment]
@pierDipi afaik @lionelvillard is rewriting that doc on adding event sources to update the process.
Issue #2970 - any updates on this yet Lionel? Will it make it in for the v0.19.0 release or should we remove this doc temporarily in the meantime?

**Review by pierDipi — APPROVED** [review]
I missed that issue, sorry!

/lgtm

**Pull request merged**